src/sections/2021/section-3.js

```diff
@@ -524,8 +524,11 @@ const Section3 = () => (
     </GridCell>
 
     <GridCell startMD="6" alignMD="end" startLG="9" className="util-margin-bottom-1xl util-margin-bottom-none@md">
-      <p>Though many respondents thought their design system(s) created debt, this did not correlate to the perception of a less successful design system.</p>
-      <p><strong>How is it that respondents could report debt being a major challenge, but it doesn’t impact perceived success?</strong></p>
+      <SmallCallout>
```

Comment:
I asked Kaitlyn / Emily about how this should be formatted. Sounds like we should separate the callout here from the heading and paragraph below. I'd put them in separate grid items and have the small callout aligned with the bottom of the chart that it is associated with. I'd also make it thinner on `md` and `lg` screens. Fig 5.1 and the following callout, heading, and paragraph is a good example of how the spacing should be handled except with a slightly different layout here.